proto/compiler.bzl

```diff
@@ -193,7 +211,7 @@ def _go_proto_compiler_impl(ctx):
 
 _go_proto_compiler = rule(
     implementation = _go_proto_compiler_impl,
-    attrs = {
+    attrs = dict({
```

Comment:
I wonder if we could use the `{} | {}` to merge the 2 dicts, instead of `dict({}, **{})`?

Reply:
It's not available on Bazel 5.4.0, which is our current minimum.

Reply:
Oh haha, I forgot about Bazel LTS still has 5.4 as supported 😓 

[One ~~day~~year more](https://www.youtube.com/watch?v=Sv-BxH3SVS8)